builds/misc/addons-release.yaml

```diff
@@ -20,11 +20,11 @@ jobs:
           name: API Proxy
           imageName: azureiotedge-api-proxy
           project: api-proxy-module
-          version: $(version)
+          version: $(Build.BuildNumber)
```

Comment:
I would leave this one as it was, since it was working. If you change this and someone actually enters a value for the "version" variable, this change would ignore it.